sdks/python/apache_beam/io/gcp/gcsio_integration_test.py

```diff
@@ -168,33 +139,6 @@ def _test_copy_batch(
   def test_copy_batch(self):
     self._test_copy_batch("test_copy_batch")
 
-  @pytest.mark.it_postcommit
```

Comment:
Can we keep these tests? They are running on real kms key (different from unit test)

Reply:
I'm not sure they're testing anything since we aren't directly managing kms keys during copying anymore. Is there another reason to keep them?

Reply:
The GCS client handles kms automatically. The default is to use the bucket kms for a new object in that bucket. If an object is copied from one bucket to another, the copy will have the new bucket's kms. The only way the new GCSIO interacts with kms (beyond retrieving keys from buckets or objects) is by allowing it to be set on a new bucket.